src/core/qgshttptransaction.h

```diff
@@ -114,6 +115,8 @@ class CORE_EXPORT QgsHttpTransaction : public QObject
 
     void networkTimedOut();
 
+    void ignoreSslErrors( const QList<QSslError> &errors );
```

Comment:
I know the documentation in QGIS Server 2.X is not very expanded, but can you add a little note with `@since` in this case?